scripts/install/opt/seagate/cortx/motr/libexec/motr_data_recovery.sh

```diff
@@ -771,16 +788,17 @@ run_becktool() {
 
             m0drlog "Running Becktool on remote node"
             # Below statement is for logging purpose
-            echo "Running '$BECKTOOL -s $SOURCE_IMAGE -d $DEST_DOMAIN_DIR/db -a $DEST_DOMAIN_DIR/stobs'" >> ${LOG_FILE}
+            echo "Running '(cd $MD_DIR/datarecovery; $BECKTOOL -s $SOURCE_IMAGE -d $DEST_DOMAIN_DIR/db -a $DEST_DOMAIN_DIR/stobs;)'" >> ${LOG_FILE}
             run_cmd_on_remote_node "bash -s" <<-EOF
-            $BECKTOOL -s $SOURCE_IMAGE -d $DEST_DOMAIN_DIR/db -a $DEST_DOMAIN_DIR/stobs
+            (cd $MD_DIR/datarecovery; $BECKTOOL -s $SOURCE_IMAGE -d $DEST_DOMAIN_DIR/db -a $DEST_DOMAIN_DIR/stobs;)
 EOF
```

Comment:
why not:
```
run_cmd_on_remote_node "(cd $MD_DIR/datarecovery; $BECKTOOL -s $SOURCE_IMAGE -d $DEST_DOMAIN_DIR/db -a $DEST_DOMAIN_DIR/stobs;)"
```

This style is used elsewhere. So, why is "bash -s" << -EOF
xxxxx
EOF
used here?

Reply:
This is done because the beck tool might crash if it runs out of memory as we have taken away the SWAP when becktool runs. In such a case the return value from becktool is over 128 and these return values are not returned by ssh to the caller, so Kanchan found this little trick on the web and it works.